docs/source/features/optimizations/activation_recomputation.rst

```diff
@@ -35,4 +35,18 @@ This is because the input sizes of softmax, dropout, and qkv dot-product attenti
 However, their recomputation cost is relatively smaller than the other linear projection layers that are linear with the hidden size square.
 
 Self-attention recomputation is hard-enabled when using FlashAttention, which is supported in Transformer Engine.
-Also, a user can use the self-attention recomputation without FlashAttention by setting ``activations_checkpoint_granularity=selective``.
\ No newline at end of file
+Also, a user can use the self-attention recomputation without FlashAttention by setting ``activations_checkpoint_granularity=selective``.
+
+Scheme of full and selective checkpointing granularity:
+
+.. image:: https://github.com/NVIDIA/NeMo/releases/download/v2.0.0rc0/asset-post-activation-recomputation-exampe-2.jpg
+    :align: center
+    :alt: activation-recomputation-example-2
+    :scale: 50%
```

Comment:
Could you double-check this -- I'm getting a resize error in RTD rendering, but maybe this will turn out fine in the final docs.